src/rtcModule/webrtcAdapter.cpp

```diff
@@ -173,7 +174,21 @@ void InputDeviceShared<webrtc::VideoTrackInterface, webrtc::VideoTrackSourceInte
     rtc::Thread *currentThread = rtc::ThreadManager::Instance()->CurrentThread();
     mSource = new rtc::RefCountedObject<webrtc::AndroidVideoTrackSource>(currentThread, env, surfaceTextureHelper, false);
     rtc::scoped_refptr<webrtc::VideoTrackSourceProxy> proxySource = webrtc::VideoTrackSourceProxy::Create(currentThread, currentThread, mSource);
-    env->CallStaticVoidMethod(applicationClass, startVideoCaptureMID, (jlong)proxySource.release(), surfaceTextureHelper);
+    if (startVideoCaptureWithParametersMID)
+    {
+        std::string widthString;
+        mOptions->constraints.GetMandatory().FindFirst(webrtc::MediaConstraintsInterface::kMinWidth, &widthString);
+        int width = std::atoi(widthString.c_str());
+        std::string heightString;
+        mOptions->constraints.GetMandatory().FindFirst(webrtc::MediaConstraintsInterface::kMinHeight, &heightString);
+        int height = std::atoi(heightString.c_str());
+        env->CallStaticVoidMethod(applicationClass, startVideoCaptureWithParametersMID, (jint)width, (jint)height, (jint)15, (jlong)proxySource.release(), surfaceTextureHelper);
```

Comment:
We could improve readability by declaring `int fps = 15;` here, instead of passing a hardcoded value.

Reply:
Solved it